renderer/reducers/channels.js

```diff
@@ -843,7 +844,10 @@ const channelSortSelector = state => state.channels.sort
 const channelSortOrderSelector = state => state.channels.sortOrder
 const filterSelector = state => state.channels.filter
 const nodesSelector = state => networkSelectors.nodes(state)
-const viewModeSelector = state => state.channels.viewMode
```

Comment:
Since `state.channels.viewMode` is no longer used, I'd suggest that we remove it from the state altogether to avoid confusion.

Reply:
@mrfelton I hope I understood your correctly. See the follow-up commit.

Reply:
Yes, that's right @bolatovumar . However, the seconds commit should have the scope `refactor` instead of `fix` since it does not fix an actual user facing bug and therefore is not something that we'd want to include in the changelog.